app/repositories/MongoJourneyAnswersRepository.scala

```diff
@@ -95,12 +97,12 @@ class MongoJourneyAnswersRepository @Inject() (mongo: MongoComponent, clock: Clo
   }
   private[repositories] def createUpsert(
     ctx: JourneyContext
-  )(fieldName: String, value: BsonValue, statusOnInsert: JourneyStatus) = {
+  )(fieldName: String, bsonValue: BsonValue, statusOnInsert: JourneyStatus) = {
     val now = Instant.now(clock)
-    val expireAt = calculateExpireAt(now)
+    val expireAt = now.atZone(ZoneOffset.UTC).truncatedTo(ChronoUnit.DAYS).plusDays(appConfig.timeToLive).toInstant
```

Comment:
is calculateExpireAt being used? if not we can delete it OR we can move this logic into that function maybe?

Reply:
Its not getting used. . I have just left it as there are still discussions happening for 28 days. They may revert it to 4 years Or a longer duration instead so left it there.  I will delete it as soon as its finalized.